.github/workflows/tests.yml

```diff
@@ -0,0 +1,77 @@
+name: EvaP Test Suite
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-18.04
+
+    container:
+      image: python:3.7
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: evap
+        ports:
+        - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    strategy:
+      matrix:
+        include:
+          - name: "Coverage"
+            command: "coverage run manage.py test && codecov -X gcov"
+          - name: "Debug mode"
+            command: "python manage.py test --debug-mode"
+          - name: "Reverse order"
+            command: "python manage.py test --reverse"
+          #- name: "Backup process"
+          #  command: "deployment/test_backup_and_restore_on_github.sh"
```

Comment:
this is not yet working, will look into it

Reply:
@He3lixxx, it looks like loading the backup starts already before the update script is done. could you have a look at it? ( https://github.com/janno42/EvaP/runs/1814767358?check_suite_focus=true)